Views/MainView.py

```diff
@@ -0,0 +1,63 @@
+from tkinter import *
+from Views.ColorButton import ColorButton
+
+class MainView(Tk):
+    class Constants:
+        title = 'Pizarra Magica'
+        height = 700
+        width = 500
+        center = N + S + E + W
+        span = 4
+        last_horizontal = None
+        last_vertical = None
+        color = "#202020"
+
+        @classmethod
+        def size(cls):
+            return "{}x{}".format(cls.width, cls.height)
+
+    def __init__(self):
+        super().__init__()
+        self.title(self.Constants.title)
+        self.geometry(self.Constants.size())
+        self.minsize(self.Constants.width, self.Constants.height)
+        self.maxsize(self.Constants.width, self.Constants.height)
+        self.__configure_ui()
+        self.__configure_grid()
+
+        self.bind("<space>", self.clean)
+
+    def __configure_grid(self):
+        self.grid_rowconfigure(0, weight=True)
+        self.grid_rowconfigure(1, minsize = self.Constants.height - 600, weight=True)
+
+        for column_index in range(0, 4):
+            self.grid_columnconfigure(column_index, minsize=self.Constants.width / 4, weight=True)
+
+    def __configure_ui(self):
+        self.__canvas = Canvas(self, width = self.Constants.width, height = self.Constants.height, bg = 'gray')
+        self.__canvas.grid(row = 0, column = 0, sticky = self.Constants.center, columnspan = self.Constants.span)
+
+        self.__blue_button = ColorButton(self, 'Blue', '#278ad1', click_handler = self.__color_chooser)
```

Comment:
mas magic numbers y strings en colores y textos. Por otro lado repites mucho codigo solo para crear botones, no seria mas facil con un for?